lib/runtime/wasmer/instance.go

```diff
@@ -264,67 +264,45 @@ func (in *Instance) Stop() {
 	}
 }
 
-// Store func
-func (in *Instance) store(data []byte, location int32) {
```

Comment:
why did we want to remove these functions?

Reply:
That was me trying to understand everything that was needed in that `Exec` method, and those tiny methods made it just hard to understand tbh.

And these 1 or 2-lines functions were only used once (in prod code, and once in a test), so I just inlined them.

So I took the EXECUTIVE DECISION to inline them pretty much 😄